### PR TITLE
Change team members and add QoL updates

### PR DESCRIPTION
**Pull request by jcayabyab**
Merging this PR into master will automatically update the team members on the live website.

QoL changes:

- Fix broken sponsors button
- Add null check if no profile picture in the CMS
- Add customizable ordering
- Add favicon
- Change website title

_Note: The website update system through the headless CMS seems to be currently broken, i.e., the website needs to be redeployed to reflect any changes in the CMS (we should look into avoiding this when creating the website redesign)._

### CODE REVIEW

File: src/components/Card.jsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ export const CardExecMember = ({ admin }) => {
   const fullName = admin.fullName || 'DEFAULT'
   const linkedIn = admin.linkedIn || 'DEFAULT'
   const position = admin.position || 'DEFAULT'
-  const img = admin.profilePicture.fluid || ''
+  const img = (admin.profilePicture && admin.profilePicture.fluid) || ''
```
Comment (tylerlam1) [inline src/components/Card.jsx:64]: Would it make more sense to just do `admin.profilePicture?.fluid` to handle the null checks?

**Review by tylerlam1 — APPROVED** [review]
looks great!

File: src/components/Card.jsx (hunk 1 above)
Comment (jcayabyab) [reply src/components/Card.jsx:64]: I wasn't sure what version this repo deploys to on CodePipeline (there's no files on it in the repo afaik and there's no `.nvm` file), and this syntax was introduced in Node 14, so I used this syntax. If you think it's worth checking out the CodePipeline deployment let me know

Comment (tylerlam1) [reply src/components/Card.jsx:64]: Good observation, let's leave it then

**Pull request merged**